src/Dfc.CourseDirectory.Core/Search/AzureCognitiveSearch/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Azure;
+using Azure.Search.Documents;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Dfc.CourseDirectory.Core.Search.AzureCognitiveSearch
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddAzureCongnitiveSearchClient<TQuery, TResult>(this IServiceCollection services, Uri endpoint, string key, string indexName)
```

Comment:
```suggestion
        public static IServiceCollection AddAzureCognitiveSearchClient<TQuery, TResult>(this IServiceCollection services, Uri endpoint, string key, string indexName)
```

Reply:
Well spotted. I'm thinking about dropping the `Cognitive` from `AzureCognitiveSearch...` - thoughts?